app/screens/channel_info/index.js

```diff
@@ -56,6 +57,7 @@ function mapStateToProps(state) {
     const currentChannelCreatorName = currentChannelCreator && currentChannelCreator.username;
     const currentChannelStats = getCurrentChannelStats(state);
     const currentChannelMemberCount = currentChannelStats && currentChannelStats.member_count;
+    let currentChannelGuestCount = currentChannelStats && currentChannelStats.guest_count;
```

Comment:
consider the fact that `guest_count` can be null cause of using the app with previous server versions

Reply:
I going to add an `|| 0` at the end. anyway, we only compare it with `> 0` so the result would be the same. Still makes sense to make it more clear right here.